### PR TITLE
August 2020 SC Data release notes

### PR DESCRIPTION
**Pull request by pcm32**
I didn't mention CoVid-19 datasets since only an additional one was added and they will be shown in the collections.

### CODE REVIEW

File: release-notes/sc/_posts/2020-05-14-10.md [hunk 1]
```diff
@@ -1,42 +1,14 @@
...
-- [scRNA-seq analysis of lung CD64-expressing mononuclear cells, patrolling and classical monocytes from steady-state C57BL/6J mice](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7678)
-- [scRNA-seq of human fetal lung primary tissues and cell cultures derived from fetal bud tip progenitors under dual SMAD treatment](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-8221)
-
+- [Single-cell RNA-seq of nasal cavity mucosa cells of mice stimulated with interferon alpha](https://www.ebi.ac.uk/gxa/sc/experiments/E-CURD-52)
+- [A single-cell atlas of the peripheral immune response to severe COVID-19](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-150728)
+- [Single cell sequencing of developing heart valves identifies novel cell populations and uncovers interstitial cell heterogeneity](https://www.ebi.ac.uk/gxa/sc/experiments/E-HCAD-18)
+- [Functional germ line stem cells do not exist in adult mammalian ovaries](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-2983)
+- [Single-cell RNA-seq of of dermal fibroblasts (supplementary - human fibroblasts stimulation with dsRNA)](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7037)
```
Comment (pcm32) [inline release-notes/sc/_posts/2020-05-14-10.md]:
```suggestion
- [Single-cell RNA-seq of dermal fibroblasts (supplementary - human fibroblasts stimulation with dsRNA)](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7037)
```

@ngeorgeebi this accession might have an extra "of" in the title in the IDF? I have corrected it here.

File: release-notes/sc/_posts/2020-05-14-10.md [hunk 2]
```diff
@@ -1,42 +1,14 @@
...
+- [Single-cell RNA-seq of dermal fibroblasts (5 human individuals - IFNB stimulation)](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7051)
+- [Single-cell RNA-seq of of human dermal fibroblasts (supplementary - different stimulation conditions of dsRNA)](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7052)
```
Comment (pcm32) [inline release-notes/sc/_posts/2020-05-14-10.md]:
```suggestion
- [Single-cell RNA-seq of human dermal fibroblasts (supplementary - different stimulation conditions of dsRNA)](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7052)
```

Could you also please check this one on the IDF @ngeorgeebi ? Thanks! Same extra "of".

**Comment by pcm32** [comment]
@ngeorgeebi besides E-CURD-52 and E-GEOD-150728 here, do any of the other studies need to be added to the COVID-19 collection? Thanks.

**Comment by pcm32** [comment]
I have added those two to the COVID-19 collection and updated fallback with that manually (as it was already migrated).

File: release-notes/sc/_posts/2020-05-14-10.md [hunk 3]
```diff
@@ -1,42 +1,20 @@
...
+- [Functional germ line stem cells do not exist in adult mammalian ovaries](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-2983)
+- [Single-cell RNA-seq of dermal fibroblasts (supplementary - human fibroblasts stimulation with dsRNA)](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7037)
+- [Single-cell RNA-seq of dermal fibroblasts (5 human individuals - IFNB stimulation)](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7051)
+- [Single-cell RNA-seq of human dermal fibroblasts (supplementary - different stimulation conditions of dsRNA)](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7052)
+- [Single cell sequencing of colorectal tumors and adjacent non-malignant colon tissue](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-8410)
 
+#### New features
+- In the experiment page, Supplymentary Information tab, a section called Resources is added for external resource links. There are resources potentiallly from ArrayExpress, GEO, ENA and EGA.
+- Now user able to download images for large tSNE plots without failure. e.g.[E-HCAD-14](https://www.ebi.ac.uk/gxa/sc/experiments/E-HCAD-14)
```
Comment (pcm32) [inline release-notes/sc/_posts/2020-05-14-10.md]:
```suggestion
- Now users are able to download images for large tSNE plots without failure. e.g.[E-HCAD-14](https://www.ebi.ac.uk/gxa/sc/experiments/E-HCAD-14)
```

File: release-notes/sc/_posts/2020-05-14-10.md [hunk 4]
```diff
@@ -1,42 +1,20 @@
...
+- [Single cell sequencing of developing heart valves identifies novel cell populations and uncovers interstitial cell heterogeneity](https://www.ebi.ac.uk/gxa/sc/experiments/E-HCAD-18)
+- [Functional germ line stem cells do not exist in adult mammalian ovaries](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-2983)
+- [Single-cell RNA-seq of dermal fibroblasts (supplementary - human fibroblasts stimulation with dsRNA)](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7037)
+- [Single-cell RNA-seq of dermal fibroblasts (5 human individuals - IFNB stimulation)](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7051)
+- [Single-cell RNA-seq of human dermal fibroblasts (supplementary - different stimulation conditions of dsRNA)](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7052)
+- [Single cell sequencing of colorectal tumors and adjacent non-malignant colon tissue](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-8410)
 
+#### New features
+- In the experiment page, Supplymentary Information tab, a section called Resources is added for external resource links. There are resources potentiallly from ArrayExpress, GEO, ENA and EGA.
```
Comment (pcm32) [inline release-notes/sc/_posts/2020-05-14-10.md]:
```suggestion
- In the experiment page, Supplementary Information tab, a section called Resources is available now for external resource links. These are resources potentially from ArrayExpress, GEO, ENA and EGA.
```

File: release-notes/sc/_posts/2020-05-14-10.md [hunk 5]
```diff
@@ -1,42 +1,20 @@
...
+- Now user able to download images for large tSNE plots without failure. e.g.[E-HCAD-14](https://www.ebi.ac.uk/gxa/sc/experiments/E-HCAD-14)
+- In the experiment page, marker gene heatmap. The marker gene symbols/IDs are converted as a hyperlink text, which transfer to the gene search result page.
```
Comment (pcm32) [inline release-notes/sc/_posts/2020-05-14-10.md]:
```suggestion
- Marker Gene Heatmap: The marker gene symbols/IDs shown as row labels are now hyperlinks, linking to the gene search result page within Single Cell Atlas for each gene.
```

**Comment by anjaf** [comment]
@pcm32 
> I didn't mention CoVid-19 datasets since only an additional one was added and they will be shown in the collections.

Maybe we should then actually highlight that there is now a "COVID-19" collection and add a link to it. Before we only had the link for mouse and human search results on the front page.

File: release-notes/sc/_posts/2020-08-21-11.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+- [Single cell sequencing of developing heart valves identifies novel cell populations and uncovers interstitial cell heterogeneity](https://www.ebi.ac.uk/gxa/sc/experiments/E-HCAD-18)
+- [Functional germ line stem cells do not exist in adult mammalian ovaries](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-2983)
+- [Single-cell RNA-seq of dermal fibroblasts (supplementary - human fibroblasts stimulation with dsRNA)](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7037)
+- [Single-cell RNA-seq of dermal fibroblasts (5 human individuals - IFNB stimulation)](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7051)
+- [Single-cell RNA-seq of human dermal fibroblasts (supplementary - different stimulation conditions of dsRNA)](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7052)
+- [Single cell sequencing of colorectal tumors and adjacent non-malignant colon tissue](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-8410)
+
+#### New features
+- In the experiment page, Supplementary Information tab, a section called Resources is available now for external resource links. These are resources potentially from ArrayExpress, GEO, ENA and EGA.
```
Comment (anjaf) [inline release-notes/sc/_posts/2020-08-21-11.md:18]: Was this feature tested and approved? I don't see any working links to GEO. Also https://www.ebi.ac.uk/gxa/sc/experiments/E-HCAD-14/supplementary-information links wrongly to ENA.

File: release-notes/sc/_posts/2020-08-21-11.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+- Now users are able to download images for large tSNE plots without failure. e.g.[E-HCAD-14](https://www.ebi.ac.uk/gxa/sc/experiments/E-HCAD-14)
+- Marker Gene Heatmap: The marker gene symbols/IDs shown as row labels are now hyperlinks, linking to the gene search result page within Single Cell Atlas for each gene.
+- In the Browse Experiments page, after checking the box for downloading experiment and click the download link, a popup window lists three checkboxes for file type selection.
```
Comment (anjaf) [inline release-notes/sc/_posts/2020-08-21-11.md]:
```suggestion
- In the Browse Experiments page, after checking the box to download the experiment and clicking the download link, a pop-up window now lets you select the file types (raw quantification, normalised counts or experimental design) to download.
```

File: release-notes/sc/_posts/2020-08-21-11.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+- [Single-cell RNA-seq of dermal fibroblasts (supplementary - human fibroblasts stimulation with dsRNA)](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7037)
+- [Single-cell RNA-seq of dermal fibroblasts (5 human individuals - IFNB stimulation)](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7051)
+- [Single-cell RNA-seq of human dermal fibroblasts (supplementary - different stimulation conditions of dsRNA)](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7052)
+- [Single cell sequencing of colorectal tumors and adjacent non-malignant colon tissue](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-8410)
+
+#### New features
+- In the experiment page, Supplementary Information tab, a section called Resources is available now for external resource links. These are resources potentially from ArrayExpress, GEO, ENA and EGA.
+- Now users are able to download images for large tSNE plots without failure. e.g.[E-HCAD-14](https://www.ebi.ac.uk/gxa/sc/experiments/E-HCAD-14)
+- Marker Gene Heatmap: The marker gene symbols/IDs shown as row labels are now hyperlinks, linking to the gene search result page within Single Cell Atlas for each gene.
```
Comment (anjaf) [inline release-notes/sc/_posts/2020-08-21-11.md]:
```suggestion
- Marker Gene Heatmap: The marker gene symbols/IDs shown as row labels now act as hyperlinks, linking to the gene search result page within Single Cell Expression Atlas.
```

File: release-notes/sc/_posts/2020-08-21-11.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+- [Functional germ line stem cells do not exist in adult mammalian ovaries](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-2983)
+- [Single-cell RNA-seq of dermal fibroblasts (supplementary - human fibroblasts stimulation with dsRNA)](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7037)
+- [Single-cell RNA-seq of dermal fibroblasts (5 human individuals - IFNB stimulation)](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7051)
+- [Single-cell RNA-seq of human dermal fibroblasts (supplementary - different stimulation conditions of dsRNA)](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7052)
+- [Single cell sequencing of colorectal tumors and adjacent non-malignant colon tissue](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-8410)
+
+#### New features
+- In the experiment page, Supplementary Information tab, a section called Resources is available now for external resource links. These are resources potentially from ArrayExpress, GEO, ENA and EGA.
+- Now users are able to download images for large tSNE plots without failure. e.g.[E-HCAD-14](https://www.ebi.ac.uk/gxa/sc/experiments/E-HCAD-14)
```
Comment (anjaf) [inline release-notes/sc/_posts/2020-08-21-11.md]:
```suggestion
- A bug was fixed to allow users to download images for large tSNE plots, e.g.[E-HCAD-14](https://www.ebi.ac.uk/gxa/sc/experiments/E-HCAD-14)
```

Comment (anjaf) [reply release-notes/sc/_posts/2020-08-21-11.md]: And maybe move that to the end of the features list.

File: release-notes/sc/_posts/2020-08-21-11.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+- [A single-cell atlas of the peripheral immune response to severe COVID-19](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-150728)
+- [Single cell sequencing of developing heart valves identifies novel cell populations and uncovers interstitial cell heterogeneity](https://www.ebi.ac.uk/gxa/sc/experiments/E-HCAD-18)
+- [Functional germ line stem cells do not exist in adult mammalian ovaries](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-2983)
+- [Single-cell RNA-seq of dermal fibroblasts (supplementary - human fibroblasts stimulation with dsRNA)](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7037)
+- [Single-cell RNA-seq of dermal fibroblasts (5 human individuals - IFNB stimulation)](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7051)
+- [Single-cell RNA-seq of human dermal fibroblasts (supplementary - different stimulation conditions of dsRNA)](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7052)
+- [Single cell sequencing of colorectal tumors and adjacent non-malignant colon tissue](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-8410)
+
+#### New features
```
Comment (anjaf) [inline release-notes/sc/_posts/2020-08-21-11.md:16]:
```suggestion
#### New features
- A new experiment collection for COVID-19 related research: [Browse COVID-19 experiments](https://www.ebi.ac.uk/gxa/sc/experiments?experimentProjects=%22Covid-19%22)
```

File: release-notes/sc/_posts/2020-08-21-11.md (hunk 6 above)
Comment (pcm32) [reply release-notes/sc/_posts/2020-08-21-11.md:18]: @lingyun1010 can you please comment on whether this was tested. We would need to find out why this experiment links wrongly to ENA and fix it for a posterior release. But if this is working wrongly, I rather not announce it.

Comment (pcm32) [reply release-notes/sc/_posts/2020-08-21-11.md:18]: @alfonsomunozpomer FYI

**Pull request merged**